pkg/services/sqlstore/sqlstore.go

```diff
@@ -473,23 +478,38 @@ func InitTestDB(t ITestDB, opts ...InitTestDBOpt) *SQLStore {
 			t.Fatalf("Failed to init test database: %v", err)
 		}
 
-		testSQLStore.Dialect = migrator.NewDialect(engine)
+		engine.DatabaseTZ = time.UTC
+		engine.TZLocation = time.UTC
 
-		// temp global var until we get rid of global vars
-		dialect = testSQLStore.Dialect
+		testSQLStore, err = newSQLStore(cfg, localcache.New(5*time.Minute, 10*time.Minute), bus.GetBus(), engine, opts...)
```

Comment:
Using `bus.GetBus()` instead of `bus.New()`. Without this have to replace bus instance when setup api integration tests.